gui/gui/inc/TGSpeedo.h

```diff
@@ -61,6 +61,9 @@ class TGSpeedo : public TGFrame, public TGWidget {
    Bool_t           fThresholdActive;     // kTRUE if glowing threhsholds are active
    Bool_t           fPeakMark;            // kTRUE if peak mark is active
    Bool_t           fMeanMark;            // kTRUE if mean mark is active
+   Int_t            fBufferSize;          // circular buffer size
+   Int_t            fBufferCount;         // circular buffer count
```

Comment:
Better use Long64_t for fBufferCount in case you remove line 419

Reply:
Well, no, it would just take a bit longer before the overflow...

Reply:
You're right.
In that case, my proposal here would not be right:
if(fBufferCount < fBufferSize)
{
fBuffer.push_back(fValue);
}
and it should be changed with sth like:
if(fBuffer.size() < fBufferCount+1)
{
fBuffer.push_back(fValue);
}